python/ray/util/spark/databricks_hook.py

```diff
@@ -54,21 +58,102 @@ def display_databricks_driver_proxy_url(spark_context, port, title):
     )
 
 
+DATABRICKS_AUTO_SHUTDOWN_POLL_INTERVAL_SECONDS = 3
+DATABRICKS_RAY_ON_SPARK_AUTOSHUTDOWN_TIMEOUT_MINUTES = (
```

Comment:
Why are we using minutes?

Reply:
Usually user should set a a long time, minute unit is more convenient.

Reply:
Got it